.github/workflows/ci.yaml

```diff
@@ -246,20 +455,21 @@ jobs:
         if: always()
         uses: actions/upload-artifact@v3
         with:
-          name: gradle_jdbc_postgresql_integration_test_reports
+          name: postgresql_15_integration_test_reports
           path: core/build/reports/tests/integrationTestJdbc
 
-  integration-test-for-jdbc-oracle:
+  integration-test-for-jdbc-oracle-18:
+    name: Oracle 18 integration test
     runs-on: ubuntu-latest
 
     services:
       oracle:
-        image: ghcr.io/scalar-labs/oracle/database:21.3.0-xe
+        image: ghcr.io/scalar-labs/oracle/db-prebuilt:18.4.0-xe
```

Comment:
For all Oracle jobs, I generated an image version that has prebuilt the database (see [here](here) for the image details ). 
The image size is significantly increased (between 10 and 15GB with the prebuilt database version vs  6GB for the mint version), so it takes longer to download, but it takes only about 1 minute to boot now.
In total, Oracle integration test jobs runtime was reduced by more or less 5 minutes.